web/src/main/java/com/graphhopper/countries/GermanySpatialRule.java

```diff
@@ -15,17 +15,18 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package com.graphhopper.routing.util.spatialrules.countries;
+package com.graphhopper.countries;
```

Comment:
Adding the web module for using the SR lookup might not be optimal but putting it into core does not make sense as core is not able to parse the json file. So we could introduce a util module and depend on core&json&osm (?)